web/src/hash_util.ts

```diff
@@ -20,12 +20,12 @@ import type {UserGroup} from "./user_groups.ts";
 import {user_settings} from "./user_settings.ts";
 import * as util from "./util.ts";
 
-export function build_reload_url(): string {
+export function build_reload_hash(): string {
```

Comment:
`get_reload_hash` might be more appropriate?

Reply:
I like that, changed.